setup.py

```diff
@@ -397,12 +397,21 @@ def get_mluops_version(file_path):
         elif (os.getenv('FORCE_NPU', '0') == '1'):
             print(f'Compiling {ext_name} only with CPU and NPU')
             try:
+                import imp
```

Comment:
The [imp](https://docs.python.org/3.11/library/imp.html#module-imp) module is deprecated in favor of [importlib](https://docs.python.org/3.11/library/importlib.html#module-importlib). Please use importlib.